features/inspect_bin.feature

```diff
@@ -11,12 +11,12 @@ Feature: Inspect Bin
   As a Collection Manager
   I want to review the contents of its bin.
 
-  Scenario:
+  Scenario: Revision summary
     Given a preserved monograph with an alternate identifier of "xyzzy:00000001"
     When the Collection Manager looks up the bin by "xyzzy:00000001"
     Then the Collection Manager sees the summary of the bin
 
-  Scenario:
+  Scenario: Revision file sets
```

Comment:
I guess you're futureproofing? but I guess I would leave off this "Revision" terminology until we discuss it more and introduce it explicitly. I'd stick to "Bin summary" and "Bin file sets"?

Reply:
These are basically spoilers for the version discussion, which leads to asking whether the code ever knows about the _bin_ (except for the gateway, as bin == OCFL object) and the application/catalog is more concerned with capturing the information of a specific OCFL *version*.